superset/examples/world_bank.py

```diff
@@ -460,12 +459,6 @@ def create_slices(tbl: BaseDatasource) -> List[Slice]:
         "meta": {"chartId": 49, "height": 50, "sliceName": "Treemap", "width": 8},
         "type": "CHART",
     },
-    "CHART-3nc0d8sk": {
-        "children": [],
-        "id": "CHART-3nc0d8sk",
-        "meta": {"chartId": 50, "height": 50, "sliceName": "Treemap", "width": 8},
-        "type": "CHART",
-    },
```

Comment:
Bycatch: this is a duplicate slice where the chart component isn't really used in the layout.